tests/tests_integration/tests_core_integration/tests_db_reader_integration/test_kafka_reader_integration.py

```diff
@@ -0,0 +1,65 @@
+import json
+import secrets
+
+import pytest
+
+from onetl.connection import Kafka
+from onetl.db import DBReader
+
```

Comment:
```suggestion
from onetl.db import DBReader

pytestmark = [pytest.mark.kafka, pytest.mark.db_connection, pytest.mark.connection]
```

Без mark.kafka CI не запустит этот тест.